### ISSUE TITLE
Customising the print loader

### ISSUE THREAD

**Issue by yjrandom** [react-pdf-viewer/react-pdf-viewer#1168]
Is there a way to customise the print loader?

I'm looking for a way to pass in a custom print loader renderer, like the default loader renderer.

Thanks!

**Comment by phuoc-ng** [comment]
I will support it soon.

**Comment by phuoc-ng** [comment]
@yjrandom 
The feature was implemented and will be available in the next v3.6.0

## API

```ts
const printPluginInstance = printPlugin({
    renderProgressBar: (numLoadedPages: number, numPages: number, onCancel: () => void) => (
        // Render the progress bar
    ),
});
```

## Demo

```tsx
const renderProgressBar = React.useCallback(
    (numLoadedPages: number, numPages: number, onCancel: () => void) => (
        <div
            style={{
                alignItems: 'center',
                background: 'rgba(0, 0, 0, .5)',
                display: 'flex',
                height: '100%',
                justifyContent: 'center',
                left: 0,
                position: 'absolute',
                top: 0,
                width: '100%',
                zIndex: 9999,
            }}
        >
            <div
                style={{
                    alignItems: 'center',
                    backgroundColor: '#fff',
                    border: '1px solid rgba(0, 0, 0, 0.3)',
                    borderRadius: '0.25rem',
                    display: 'flex',
                    flexDirection: 'column',
                    padding: '1.5rem',
                    width: '20rem',
                }}
            >
                <div style={{ marginBottom: '1rem' }}>
                    Preparing {numLoadedPages}/{numPages} pages ...
                </div>
                <div style={{ marginBottom: '1rem' }}>
                    <Spinner />
                </div>
                <Button onClick={onCancel}>Cancel</Button>
            </div>
        </div>
    ),
    []
);

const printPluginInstance = printPlugin({
    renderProgressBar,
});
```

https://user-images.githubusercontent.com/57786711/174435440-950b4c8d-90ce-4278-9773-a5b523e33e7e.mp4

**Comment by phuoc-ng** [comment]
[v3.6.0](https://react-pdf-viewer.dev/updates/) is released. Please upgrade.
I also added an [example](https://react-pdf-viewer.dev/examples/customize-the-progress-bar-when-preparing-pages-to-print/) for this.
